experiments/kube-aws/az-down/experiment/az-down.go

```diff
@@ -0,0 +1,329 @@
+package experiment
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	litmusLIB "github.com/litmuschaos/litmus-go/chaoslib/litmus/az-down/lib"
+	"github.com/litmuschaos/litmus-go/pkg/clients"
+	"github.com/litmuschaos/litmus-go/pkg/cloud/aws"
+	"github.com/litmuschaos/litmus-go/pkg/events"
+	experimentEnv "github.com/litmuschaos/litmus-go/pkg/kube-aws/az-down/environment"
+	experimentTypes "github.com/litmuschaos/litmus-go/pkg/kube-aws/az-down/types"
+	"github.com/litmuschaos/litmus-go/pkg/log"
+	"github.com/litmuschaos/litmus-go/pkg/probe"
+	"github.com/litmuschaos/litmus-go/pkg/result"
+	"github.com/litmuschaos/litmus-go/pkg/status"
+	"github.com/litmuschaos/litmus-go/pkg/types"
+	"github.com/sirupsen/logrus"
+	"math/rand"
+)
+
+// returns all AZ's instances are located in
+func getAzRangeOfInstances([]*experimentTypes.InstanceDetails) ([]string) {
+	// TODO check all instances and return list of zones instances are located in
+	return []string{}
```

Comment:
Should this be actually returning the az's? I can't see where they are supplied anywhere else.

Reply:
Good spot, that seems to have fallen out during rebase :+1: